docs/book/src/cronjob-tutorial/testdata/project/dist/chart/templates/manager/manager.yaml

```diff
@@ -23,9 +23,16 @@ spec:
         spec:
             containers:
                 - args:
+                    {{- if .Values.metrics.enable }}
                     - --metrics-bind-address=:8443
-                    - --leader-elect
+                    {{- else }}
+                    # Bind to :0 to disable the controller-runtime managed metrics server
+                    - --metrics-bind-address=:0
```

Comment:
Ensure that we properly disable the metrics server in c+r when it is not enabled
(more safe)